web/src/components/ListItem.tsx

```diff
@@ -58,7 +58,7 @@ const ListItem = ({ path, title, thumbnail, thumbnailSize, children, Icon }: Lis
       {!!thumbnail && <Thumbnail alt='' src={thumbnail} $thumbnailSize={thumbnailSize} />}
       <Description>
         <TitleRow>
-          <Title dir='auto'>{title}</Title>
+          <Title>{title}</Title>
```

Comment:
:thought_balloon: Could you explain a little more why this breaks the the layout only for non-recurring events? This seems very odd to me...

Reply:
I can not explain why I guess due to duplication maybe. At least setting this is not needed at this place